### PR TITLE
gave responsive style to the pages

### PR DESCRIPTION
**Pull request by rbarzi**

### CODE REVIEW

File: hint.css [hunk 1]
```diff
@@ -35,12 +36,49 @@ html, body {
...
 
-.back-btn{
+.back-btn {
   display: flex;
   margin: 20px auto;
   color: black; 
+  width: 100%;
+}
+
+.back-btn a {
```
Comment (mre1ght) [inline hint.css:47]: please devide class .back-btn and a .
```
.back-btn{
}
a {
}
```

File: hint.css [hunk 2]
```diff
@@ -35,12 +36,49 @@ html, body {
...
+  }
+
+  .title {
+    color: olive;
+    text-align: center;
+    font-size: 2em;
+    margin-bottom: 20px;
+  }
+
+  .back-btn {
```
Comment (mre1ght) [inline hint.css:81]: This .back-btn should be included in before.

File: index.html [hunk 3]
```diff
@@ -13,14 +13,14 @@ <h1>Scheme Dojo Portal</h1>
...
+      <p class="label-tag" >Question</p>
       <p class="question-tag">(car (cdr '(apple (banana orange))))</p> 
     </div>
     <div class="input-container">
-      <label class="label-tag">Answer</label>
+      <p class="label-tag">Answer</p>
       <div class="input-box">
         <input class="input-tag" id="input-tag" type="text" placeholder="Input your answer and then press enter button" /> 
-        <button class="enter-btn" id="enter-btn">Enter</button>
+        <button class="enter-btn" id="enter-btn" >Enter</button>
```
Comment (mre1ght) [inline index.html]: Please remove blank.
After
` <button class="enter-btn" id="enter-btn">Enter</button>`

File: index.html [hunk 4]
```diff
@@ -13,14 +13,14 @@ <h1>Scheme Dojo Portal</h1>
       <img class="img" src="./image/lambda.png" />
     </div>
     <div>
-      <label class="label-tag">Question</label>
+      <p class="label-tag" >Question</p>
```
Comment (mre1ght) [inline index.html]: Please remove blank.
After
`<p class="label-tag">Question</p>`

File: index.css [hunk 5]
```diff
@@ -12,45 +17,76 @@ h1 {
...
+  max-width: 100%;
+  height: auto;
 }
 
 .img {
   height: 150px;
+  max-width: 60%;
 }
 
+@media (min-width: 800px) {
```
Comment (mre1ght) [inline index.css:71]: please add comment before.
Example
```
/* When viewing from a mobile phone*/
@media (min-width: 800px) {
```

File: index.css [hunk 6]
```diff
@@ -12,45 +17,76 @@ h1 {
...
 .label-tag {
   color: olive;
-  font-size: 16px;
+  font-size: 1.75em;
+  text-align: center;
+  margin: 2%;
 }
 
+
 .image {
```
Comment (mre1ght) [inline index.css]: This class name .image and .img will make us confused.
The .image class is responsible for the CSS around the .img.
How about `.img-around` ?
If you have a good name that's OK.

File: index.css [hunk 7]
```diff
@@ -12,45 +17,76 @@ h1 {
...
+  margin-top: 1.25em;
+  width: auto;
 }
 
 .input-box {
-  height: 40px;
+  height: auto;
   display: flex;
+  flex-direction: column;
 }
```
Comment (mre1ght) [inline index.css]: Please delete one line.

File: index.css [hunk 8]
```diff
@@ -12,45 +17,76 @@ h1 {
...
 
 .input-tag {
-  height: 34px;
-  width: 80%;
-  font-size: 18px;
+  height: 3em;
+  width: auto;
+  font-size: 0.75em;
   
 }
```
Comment (mre1ght) [inline index.css:39]: Please insert one line.

File: index.css [hunk 9]
```diff
@@ -12,45 +17,76 @@ h1 {
...
+  margin-left: 0;
 }
 
 .label-tag {
   color: olive;
-  font-size: 16px;
+  font-size: 1.75em;
+  text-align: center;
+  margin: 2%;
 }
```
Comment (mre1ght) [inline index.css:56]: Please delete one line.

File: hint.css [hunk 10]
```diff
@@ -17,14 +17,15 @@ html, body {
 .title {
```
Comment (mre1ght) [inline hint.css:17]: The index.css uses the `h1 `class name. 
However, the hint.css uses the `.title `class name. 
Please use one of them.

**Review by mre1ght — CHANGES_REQUESTED** [review]

**Comment by azamiya** [comment]
# ADA REVIEW

- Eight gonna teach you how to debug page using phone.

- index page design is almost okay. one thing, when I see this page with iPhone, button style was different
IMG_5678

- hint page's button style is OK

IMG_5679

File: hint.css (hunk 1 above)
Comment (mre1ght) [reply hint.css:47]: This is my fault.
you don't have to fix this.

**Review by mre1ght — APPROVED** [review]

**Pull request merged**